### DISCUSSION TITLE
Nextjs with redux toolkit does not require next-redux-wrapper, how?

### DISCUSSION THREAD

**Discussion by robertwt7** [vercel/next.js#29639] [Help]
Hey guys,

This might be a dumb question but I can't get my head around it. I've been using nextjs with redux with [next-redux-wrapper](https://github.com/kirill-konshin/next-redux-wrapper) for a while now. However since I started using redux toolkit, I realise that nextjs team has created an [example](https://github.com/vercel/next.js/tree/canary/examples/with-redux) of integrating both tools. It doesn't use `next-redux-wrapper` at all. But i'm confused, without it, how does it work? How does the SSR populate the store before giving it back to the client?

can anyone help me explain? I'm very curious..

Thanks

**Comment by LeonWehrhahn** [comment]
Wondering the same. I have followed their example, but when navigating to a page [id].tsx. The store state is different compared to the previous page.
